libraries/networking/src/AssetUpload.h

```diff
@@ -16,6 +16,8 @@
 
 #include <QtCore/QObject>
 
+#include <cstdint>
```

Comment:
Aren't we supposed to use `stdint.h` ?
I actually don't know the difference between the 2, but that sounds like a C backward compatibility header.